src/shapes/Cylinder.cpp

```
@@ -53,6 +45,21 @@ Raytracer::Cylinder::hits(const Raytracer::Ray &ray) const {
   return {0.0, _color, this};
 }
 
+Math::Vector3D Raytracer::Cylinder::getNormal(const Math::Point3D &point) const {
```

Cppcheck (2.10-2)

[style] The function 'getNormal' is never used.

(unusedFunction)